pkg/database/database.go

```diff
@@ -209,7 +210,7 @@ func (db *Database) performResourceQuery(ctx context.Context, pq *paramQuery) ([
 	lastRow := parsedRows[len(parsedRows)-1]
 
 	for _, parsedRow := range parsedRows {
-		r, err := models.UnstructuredFromByteSlice([]byte(parsedRow.Bytes))
+		r, err := models.UnstructuredFromByteSlice([]byte(parsedRow.Resource))
```

Comment:
If the type of `Resource` is []byte no conversion is needed

```suggestion
		r, err := models.UnstructuredFromByteSlice(parsedRow.Resource)
```

Reply:
updated!